src/index.ts

```diff
@@ -808,6 +811,20 @@ class ForkTsCheckerWebpackPlugin {
     return function noopEmitCallback() {};
   }
 
+  private getLoggerMessage(
```

Comment:
Thanks for this change - could I request one more tweak to avoid confusion please?

The name and the return type of the method suggests a logger message is passed back. In reality no value is ever returned I think? Could this be tweaked to make intention a bit clearer please?

Reply:
Oops yes, you're totally right. Will update.